Readme.md

```diff
@@ -74,34 +124,53 @@ Cloning...Succeeded
 Fetching commits and trees from origin (no cache server)...Succeeded
 Configuring Watchman...Succeeded.
 Validating repo...Succeeded
+```
+
+Then, navigate into the repository's `src` directory.
 
+```sh
 $ cd ForTests/src
 $ ls
```

Comment:
I'm wondering if it would help to make this `ls -a` so we see the `.git` dir too ??